libs/output-mapping/src/Writer/Helper/PrimaryKeyHelper.php

```diff
@@ -32,71 +30,4 @@ function ($key) {
             ),
         );
     }
-
-    public static function modifyPrimaryKeyDecider(
-        LoggerInterface $logger,
-        array $currentTablePrimaryKey,
-        array $newTableConfigurationPrimaryKey,
-    ): bool {
-        $configPK = self::normalizeKeyArray($logger, $newTableConfigurationPrimaryKey);
-        if (count($currentTablePrimaryKey) !== count($configPK)) {
-            return true;
-        }
-        $currentTablePkColumnsCount = count($currentTablePrimaryKey);
-        if (count(array_intersect($currentTablePrimaryKey, $configPK)) !== $currentTablePkColumnsCount) {
-            return true;
-        }
-        return false;
-    }
-
-    public static function modifyPrimaryKey(
-        LoggerInterface $logger,
-        Client $client,
-        string $tableId,
-        array $tablePrimaryKey,
-        array $configPrimaryKey,
-    ): void {
-        $logger->warning(sprintf(
-            'Modifying primary key of table "%s" from "%s" to "%s".',
-            $tableId,
-            join(', ', $tablePrimaryKey),
-            join(', ', $configPrimaryKey),
-        ));
-        if (self::removePrimaryKey($logger, $client, $tableId, $tablePrimaryKey)) {
-            // modify primary key
-            try {
-                if (count($configPrimaryKey)) {
-                    $client->createTablePrimaryKey($tableId, $configPrimaryKey);
-                }
-            } catch (Throwable $e) {
-                // warn and try to rollback to original state
-                $logger->warning(
-                    "Error changing primary key of table {$tableId}: " . $e->getMessage(),
-                );
-                if (count($tablePrimaryKey) > 0) {
-                    $client->createTablePrimaryKey($tableId, $tablePrimaryKey);
-                }
-            }
-        }
-    }
-
-    private static function removePrimaryKey(
-        LoggerInterface $logger,
-        Client $client,
-        string $tableId,
-        array $tablePrimaryKey,
-    ): bool {
-        if (count($tablePrimaryKey) > 0) {
-            try {
-                $client->removeTablePrimaryKey($tableId);
-            } catch (Throwable $e) {
-                // warn and go on
-                $logger->warning(
-                    "Error deleting primary key of table {$tableId}: " . $e->getMessage(),
-                );
-                return false;
-            }
-        }
-        return true;
-    }
```

Comment:
K ničemu se nepoužívá. V rámci staršího refactoringu se to přesunulo do `TableStructureModifier`